composer.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "storj/uplink",
-    "description": "Decentralized object storage on Tardigrade",
+    "description": "Storj Decentralized Object Storage client",
```

Comment:
I think `Storj DCS client` would be enough.

Reply:
Done